docs/contributing/docs-contributions.md

```diff
@@ -103,6 +103,20 @@ It can be necessary to change a heading within the docs. It's important to note
 - Determine the URL you're looking for. `Changing headers` is linked with a URL ending in `changing-headers`, `Docs renaming instructions` becomes `docs-renaming-instructions`, etc.
 - Update all instances of the old URL to your new one. [Find and replace](https://code.visualstudio.com/docs/editor/codebasics#_search-across-files) in VS Code can help. Check that the context of the original link reference still makes sense with the new one.
 
+## Adding embedded GraphQL examples
+
+There are embedded examples in a few places in the docs (like the [GraphQL Reference guide](https://www.gatsbyjs.org/docs/graphql-reference/)) as a way to provide a working version of a query demonstrated or discussed in a guide. In the specific example of the GraphQL Query Options Reference page, these examples of the GraphiQL interface show how data can be queried from Gatsby's data layer.
+
+To write a new GraphQL example, a Codesandbox project with a Gatsby site can be opened at its server container link like `https://<project-id>.sse.codesandbox.io/` (for example: [https://711808k40x.sse.codesandbox.io/](https://711808k40x.sse.codesandbox.io/)). Because Codesandbox is running a Gatsby site in [`develop` mode instead of `build` mode](/docs/overview-of-the-gatsby-build-process/) you can navigate to GraphiQL by adding `/___graphql` to the link. Write an example query, and when you have a query you are satisfied with, the query fields and names will be saved as URL parameters so you can share the link. Copy the URL and use it as the `src` of an iframe:
```

Comment:
```suggestion
To write a new GraphQL example, a Codesandbox project with a Gatsby site can be opened at its server container link, for example: [https://711808k40x.sse.codesandbox.io/](https://711808k40x.sse.codesandbox.io/). Because Codesandbox is running a Gatsby site in [`develop` mode instead of `build` mode](/docs/overview-of-the-gatsby-build-process/) you can navigate to GraphiQL by adding `/___graphql` to the link. Write an example query, and when you have a query you are satisfied with, the query fields and names will be saved as URL parameters so you can share the link. Copy the URL and use it as the `src` of an iframe:
```